mappings/net/minecraft/client/network/ClientPlayNetworkHandler.mapping

```diff
@@ -82,11 +90,20 @@ CLASS dbf net/minecraft/client/network/ClientPlayNetworkHandler
 	METHOD a onScoreboardObjectiveUpdate (Lmk;)V
 	METHOD a onEntityPassengersSet (Lml;)V
 	METHOD a onTeam (Lmm;)V
+	METHOD a (Lmn;)V
```

Comment:
I don't know if it's worth mapping this and the above method too? We've names for the packets now, after all :).